client/src/i18n/en/subsidy.json

```diff
@@ -1,7 +1,7 @@
 {"SUBSIDY":{"ADDING_SUBSIDY":"Adding subsidy",
 "ADD_SUBSIDY":"Add Subsidy",
 "ALL_SUBSIDIES":"All Subsidies",
-"DESCRIPTION":"This module allows to create a subsidy for a particular billing",
-"NO_SUBSIDY":"No subsidy have been registered",
+"DESCRIPTION":"This module allows you to create a subsidy for a particular billing",
```

Comment:
Instead of "for a particular billing", it's probably better to leave it off entire end of this.  Subsidies can be tied to various patient groups, debtor groups, and the like.  So we'll just leave this as "This module allows you to create a subsidy."